code/client/installhelper

```diff
@@ -0,0 +1,372 @@
+#!/usr/local/munki/munki-python
+# encoding: utf-8
+#
+# Copyright 2011-2024 Greg Neagle.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+installhelper
+
+Created 2024-02-28.
+
+A helper tool for handling launchd items etc when installing Munki
+
+"""
+from __future__ import absolute_import
+
+# standard libs
+import glob
+import os
+import subprocess
+import time
+import sys
+
+# our libs
+from munkilib import munkilog
+from munkilib import osutils
+from munkilib import prefs
+from munkilib import FoundationPlist
+
+
+APPUSAGE_LAUNCH_AGENT = '/Library/LaunchAgents/com.googlecode.munki.app_usage_monitor.plist'
+APPUSAGE_LAUNCH_DAEMON = '/Library/LaunchDaemons/com.googlecode.munki.appusaged.plist'
+CHECK_TIME = '10'
```

Comment:
Why is this a string? later it's used in `time.sleep(CHECK_TIME)`

Reply:
Removed in https://github.com/macmule/munki/commit/3a9302164f2ead8103e32ae94278de743c8dae22